tests/gold_tests/remap/conf_remap_float.py

```diff
@@ -0,0 +1,52 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import os
+
+Test.Summary = '''
+Test command: sudo traffic_ctl config describe proxy.config.http.background_fill_completed_threshold (YTSATS-3309)
```

Comment:
It should not be necessary to `sudo` if TS is started up as a normal user.  Susan also says that `config describe` shouldn't need `sudo` in any case.

Reply:
Removed sudo. I think that was a typo.